terraform/modules/application/load_balancer.tf

```diff
@@ -0,0 +1,36 @@
+resource "aws_lb_target_group" "this" {
+  name        = var.prefix
+  port        = var.application_port
+  protocol    = "HTTP"
+  vpc_id      = var.vpc_id
+  target_type = "ip"
+
+  health_check {
+    healthy_threshold   = "3"
+    interval            = "30"
+    protocol            = "HTTP"
+    matcher             = "204"
+    timeout             = "3"
+    path                = "/health"
+    unhealthy_threshold = "2"
+  }
+}
+
+resource "aws_lb_listener_rule" "forward_cloudfront" {
+  count = var.initial_create ? 0 : 1

```

Comment:
I don't think this is necessary: on an initial creation, we shouldn't be creating resources in the application module at all - unless something in this PR has changed that, but if so I'm worried that that means the review app setup won't work.